WTWSMS/common/religions/00_religions.txt

```diff
@@ -4378,14 +4439,14 @@ pagan_group = {
 		heresy_icon = 85
 		
 		color = { 0.5 0.05 0.0 }
-		#parent = hellenic_pagan
+		parent = hellenic_pagan

```

Comment:
So the Mastimani should take over and appear in Hellenic provinces due to low moral authority?